src/webui/app/roles/roles.html

```diff
@@ -41,11 +45,13 @@
       <td class="begin-group-column">{{role.resources.cpus | decimalFloat}}</td>
       <td>{{role.resources.gpus | decimalFloat}}</td>
       <td>{{role.resources.mem * (1024 * 1024) | dataSize}}</td>
+      <td>{{role.resources.network_bandwidth | bandwidth}}</td>
       <td class="end-group-column">{{role.resources.disk * (1024 * 1024) | dataSize}}</td>
 
       <td class="begin-group-column">{{(role.quota.guarantee.cpus | decimalFloat) || "-"}}</td>
       <td>{{(role.quota.guarantee.gpus | decimalFloat) || "-"}}</td>
       <td>{{(role.quota.guarantee.mem * (1024 * 1024) | dataSize) || "-"}}</td>
+      <td>{{(role.quota.guarantee.network_bandwidth | 0) || "-"}}</td>
```

Comment:
0 is not a filter, I'm not sure `| 0` will work. Maybe more something like `|| 0` instead. We need to check IRL.